src/angular/projects/spark-core-angular/src/lib/spark-core-angular.module.ts

```diff
@@ -37,6 +37,8 @@ import { SparkOrderedListModule } from './components/sprk-ordered-list/sprk-orde
 import { SparkPaginationModule } from './components/sprk-pagination/sprk-pagination.module';
 import { SparkPromoModule } from './components/sprk-promo/sprk-promo.module';
 import { SparkStackModule } from './components/sprk-stack/sprk-stack.module';
+import { SparkStepperModule } from './components/sprk-stepper/sprk-stepper.module';
```

Comment:
Since we are only releasing vanilla, we'll wanna remove the imports here that I added when i did the angular component.